config/config.metricscheck.default.json

```diff
@@ -0,0 +1,16 @@
+{
+    "LogSettings": {
+        "EnableConsole": true,
+        "ConsoleLevel": "INFO",
+        "ConsoleJson": true,
+        "EnableFile": true,
+        "FileLevel": "INFO",
+        "FileJson": true,
+        "FileLocation": "metricscheck.log"
+    },
+    "PrometheusConfiguration": {
+        "PrometheusURL": "http://localhost:9090",
+        "MetricsUpdateIntervalInMS": 1000,
+        "HealthcheckUpdateIntervalInMS": 60000
+    }
+}
```

Comment:
I suggest we rename this to `config.metricswatcher.json` to keep consistency with the command that makes use of this.